src/main/scala/li/cil/oc/Settings.scala

```diff
@@ -193,7 +193,13 @@ class Settings(val config: Config) {
   val robotTurnCost = config.getDouble("power.cost.robotTurn") max 0
   val robotMoveCost = config.getDouble("power.cost.robotMove") max 0
   val robotExhaustionCost = config.getDouble("power.cost.robotExhaustion") max 0
-  val wirelessCostPerRange = config.getDouble("power.cost.wirelessCostPerRange") max 0
+  val wirelessCostPerRange = Array(config.getDoubleList("power.cost.wirelessCostPerRange"): _*) match {
+    case Array(tier1, tier2) =>
+      Array((tier1: Double) max 0.0, (tier2: Double) max 0.0)
+    case _ =>
+      OpenComputers.log.warn("Bad number of wireless card energy costs, ignoring.")
+      Array(0.05, 0.05)
```

Comment:
i suppose this is how you're handling "migration" of old configs

Reply:
Wouldn't this lose the value already set? That doesn't sound like a good idea...

Reply:
With the current config patch system, yes it would. There is precedent for that, I think some geolyzer values and power conversion rates were changed before. However, since the value is becoming an array, I might be able to update the config patcher to optionally  include preexisting values .

Reply:
That sounds like a good idea.